userService.js

```diff
@@ -0,0 +1,130 @@
+const mssql = require("mssql/msnodesqlv8");
+const dbConfig = require("./dbconfig");
+const sqlHelper = require("./userServiceSQLHelper");
+
+const pool = new mssql.ConnectionPool(dbConfig.config);
+
+function paginateUsers(req, res) {
```

Comment:
Application layers are broken. Services should know nothing about transport and data layers. 
`req` and `res` objects should not be past to services, SQL should not located in services as well

Reply:
Could you please clarify what do you mean? What layers?

Reply:
Fixed the transport and data layers. (I think)